react/Header/StructuredDataSchema/generate-structured-data-schema.js

```diff
@@ -0,0 +1,53 @@
+const auOrgSchema = {
+  socialLinks: [
+    'https://www.instagram.com/seekau/',
+    'https://twitter.com/seekjobs',
+    'https://plus.google.com/+seekau'
+  ],
+  telephone: '+61-1300-658-700',
+  url: 'https://www.seek.com.au'
+};
+
+const nzOrgSchema = {
+  socialLinks: [
+    'https://www.instagram.com/seeknz/',
+    'https://twitter.com/seekjobsnz',
+    'https://plus.google.com/+seeknz'
+  ],
+  telephone: '+64-0508-733-569',
+  url: 'https://www.seek.co.nz/'
+};
+
+const commonSocialLinks = [
+  'https://www.facebook.com/SEEK/',
+  'https://en.wikipedia.org/wiki/Seek_Limited',
+  'https://www.youtube.com/user/SEEKJobs',
+  'https://au.linkedin.com/company/seek'
+];
+
+const generateOrgSchema = (locale = 'AU') => {
+  const { telephone, socialLinks, url } = locale === 'NZ' ? nzOrgSchema : auOrgSchema;
+
+  return {
+    '@context': 'http://schema.org',
+    '@type': 'Organization',
+    url,
+    legalName: 'Seek Limited',
+    logo: 'https://www.seek.com.au/content/images/logos/seek-logo-positive.svg',
```

Comment:
Do we know who officially owns this URL, how it's managed, what the support policy is, etc? Might need to take this conversation offline.